src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private String filePath;
+    private PrintWriter storeInFile;
+    private boolean filePresent = false;
+    private File fileCreation;
```

Comment:
I'm not sure if I like how this variable is named, as it feels not very intuitive and self explanatory to me.